libs/core/src/lib/feed-input/directives/feed-input-button.directive.ts

```diff
@@ -0,0 +1,22 @@
+import { Directive, ElementRef, HostBinding } from '@angular/core';
+
+@Directive({

```

Comment:
Missing the typedocs explaining what this directive is doing. This will go to the API section of the documentation.